email_contact/main.go

```diff
@@ -89,27 +120,43 @@ func buildEmail(r events.APIGatewayProxyRequest) (*ses.SendEmailInput, error) {
 				Data:    aws.String(bodyRequest.Subject),
 			},
 		},
-		Source:           aws.String(getSender()),
+		Source:           aws.String(sender),
 		ReplyToAddresses: []*string{aws.String(bodyRequest.Email)},
 	}, nil
 
 }
 
-//email used for sender
-func getSender() string {
-	//TO DO obtain address from the parameter store
-	return "hello@collaction.org"
-}
+func isValid(input string, value string) error {
+
+	switch input {
+	case "email":
+		_, err := mail.ParseAddress(value)
+		return err
+	case "subject":
+		if len(value) > max_subject_length {
+			return errors.New("email subject is more than " + fmt.Sprint(max_subject_length) + " characters")
+		}
+	case "mesage":
+		if len(value) > max_message_length {
+			return errors.New("email message is more than " + fmt.Sprint(max_message_length) + " characters")
+		}
+	}
```

Comment:
The default case should not be an automatic pass, since something like the above typo would result in no checks being performed. An automatic fail would maybe be more appropriate:
Something like `default: return errors-New("unknown field")`